### PR TITLE
Feat/docs layout

### PR DESCRIPTION
**Pull request by sstraatemans**
## Reason:
Creates the 3 mayor layouts needed in the app.
Responsive

Closes [[issue link]](https://app.asana.com/0/1202676524474548/1204358202835555/f)

## Check off the following:

- [X] I have reviewed my changes and run the appropriate tests.
- [X] I have have run `rush change` to add the appropriate change logs.
- [X] I have added/edited docs / tests.
- [ ] I have added tutorials.
- [ ] I have double checked and DEFINITELY added docs.

### CODE REVIEW

File: packages/apps/kadena-docs/src/components/Layout/Full/Full.tsx [hunk 1]
```diff
@@ -1,30 +1,55 @@
...
 
+const StickyAsideWrapper = styled('div', {
+  position: 'fixed',
+  display: 'flex',
+  top: '$20',
+});
+
+const StickyAside = styled('div', {
+  position: 'sticky',
+  top: '40px',
```
Comment (eileenmguo) [inline packages/apps/kadena-docs/src/components/Layout/Full/Full.tsx:20]: Can use a token for this

File: packages/apps/kadena-docs/src/components/Layout/Full/Full.tsx [hunk 2]
```diff
@@ -1,30 +1,55 @@
...
     <>
       <Content id="maincontent">
-        <ul>{headers.map(renderListItem)}</ul>
-        {children}
+        <Article ref={docRef}>{children}</Article>
+
+        <Aside>
+          <StickyAsideWrapper>
+            <StickyAside>
+              <Heading as="h6" bold={true} transform="uppercase">
```
Comment (eileenmguo) [inline packages/apps/kadena-docs/src/components/Layout/Full/Full.tsx:46]: nit: bold is true by default on headings so you can remove this if you want to clean up props

File: packages/apps/kadena-docs/src/components/Layout/Full/style.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  listStyle: 'none',
+  margin: 0,
+  padding: '0',
+
+  '& li::before': {
+    content: '∙',
+    color: '$primaryHighContrast',
+    fontWeight: '$bold',
+    display: 'inline-block',
+    width: '$sizes.$4',
```
Comment (eileenmguo) [inline packages/apps/kadena-docs/src/components/Layout/Full/style.ts:16]: I don't think this works? Width will automatically use size tokens so you can just put `$4` but if you want to specify the group you don't need to add the period `$sizes$4`

File: packages/apps/kadena-docs/src/components/Layout/components/Footer/styles.ts [hunk 4]
```diff
@@ -10,3 +12,26 @@ export const StyledFooter: StyledComponent<'footer'> = styled('footer', {
...
+  flex: 1,
+  width: '100%',
+  justifyContent: 'space-around',
+  alignContent: 'center',
+  margin: '$2 0',
+
+  '@md': {
+    flex: 'auto',
+  },
+});
```
Comment (eileenmguo) [inline packages/apps/kadena-docs/src/components/Layout/components/Footer/styles.ts:27]: Shouldn't something like this use the stack component?

File: packages/libs/react-components/src/styles/colors/index.ts [hunk 5]
```diff
@@ -26,6 +26,8 @@ export const colors = {
   errorAccent: '#FF3338',
   errorSurface: '#FFDAD8',
   errorContrast: '#410006',
+
+  primarySurfaceInverted: '#00498F',
```
Comment (eileenmguo) [inline packages/libs/react-components/src/styles/colors/index.ts:30]: did you mean to leave these in?

Comment (sstraatemans) [reply packages/libs/react-components/src/styles/colors/index.ts:30]: whoops...thought i removed it

File: packages/apps/kadena-docs/src/components/Layout/Full/style.ts (hunk 3 above)
Comment (sstraatemans) [reply packages/apps/kadena-docs/src/components/Layout/Full/style.ts:16]: right. fixing it

**Review by isaozler — APPROVED** [review]

File: packages/apps/kadena-docs/src/utils/tests/getLayout.test.ts [hunk 6]
```diff
@@ -7,9 +7,9 @@ describe('utils getLayout', () => {
   });
   test('should return layout Full-Component when arg is anything', () => {
     expect(getLayout('full').displayName).toEqual('Full');
-    expect(getLayout('he-man').displayName).toEqual('Full');
   });
   test('should return layout Landing-Component when arg is "landing"', () => {
     expect(getLayout('landing').displayName).toEqual('Landing');
+    expect(getLayout('he-man').displayName).toEqual('Landing');
```
Comment (eileenmguo) [inline packages/apps/kadena-docs/src/utils/tests/getLayout.test.ts:13]: what is "he-man"?

**Review by eileenmguo — APPROVED** [review]
Would suggest taking a look at the changes logs to make sure there aren't duplicates from other PRs

**Pull request merged**